rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/qos_converter.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2018 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I had previously added `qos_converter` and made a mistake in the Copyright statement. This corrects that.